### ISSUE TITLE
Failed to compile. Module not found: Can't resolve 'drizzle'

### ISSUE THREAD

**Issue by hannahk0123** [ConsenSys-archive/drizzle-legacy#241]
I am working on GETTING STARTED WITH DRIZZLE AND REACT and in step [Set Up the Store](https://www.trufflesuite.com/tutorials/getting-started-with-drizzle-and-react#setup-the-store). localhost:3000 opened correctly, but whenever I add a given code to client/src/index.js, error below comes up. 

**Failed to compile.
./src/index.js
Module not found: Can't resolve 'drizzle' in '/Users/****/drizzle-react-tutorial/client/src'**

The code is
<img width="641" alt="스크린샷 2019-08-20 오전 11 04 02" src="https://user-images.githubusercontent.com/53050856/63359177-48e50980-c33a-11e9-8d5b-b8e9c32ff27e.png">

I am not sure why this error appears. Please help!
Thanks!

**Comment by cds-amal** [comment]
@hanhan0123 this could happen If drizzle isn't installed in the **_client_** folder. Can you verify it was installed there?

**Comment by hannahk0123** [comment]
@cds-amal Yes, It is installed. location: /Users/****/drizzle-react-tutorial.
<img width="404" alt="스크린샷 2019-08-21 오전 9 11 13" src="https://user-images.githubusercontent.com/53050856/63434925-fb2cd780-c3f3-11e9-9021-fc57d74ee8f0.png">

**Comment by cds-amal** [comment]
@hanhan0123, I'm unable to tell if drizzle is installed in the correct location from the image. You can verify this by the existence of this folder: **_/Users/****/drizzle-react-tutorial/client/node_modules/drizzle_**. 

If that path doesn't exist, you can fix by running `npm install drizzle` from the client folder (/Users/****/drizzle-react-tutorial/client), and restarting your app/dev server. 

If that path exists, (drizzle is installed correctly), can you please share a link to a repo so we can investigate further?

Thanks!

**Comment by hannahk0123** [comment]
@cds-amal Oh, I am sorry. I misread and thought you were asking to verify if the client folder exists. 
No, drizzle is not installed in Users/****/drizzle-react-tutorial/client/node_modules/drizzle.

So, here's what I did. 
If I run **npm install drizzle**,  this error shows.

<img width="623" alt="npm install drizzle" src="https://user-images.githubusercontent.com/53050856/63450893-093d2100-c411-11e9-94d0-73b061fe05bb.png">

When I go to that link in message, it says use **npm i tar instead** and I did. 
Then, **npm start** opens up localhost:3000 automatically and successfully. 
However, when I add the given code to client/src/index.js as it says in this step [Set up the Store](client/src/index.js), this message below comes up. 

Failed to compile.
./src/index.js
Module not found: Can't resolve 'drizzle' in '/Users/****/drizzle-react-tutorial/client/src'

I really appreciate your help!
